include/d/actor/d_a_hitobj.h

```diff
@@ -2,10 +2,29 @@
 #define D_A_HITOBJ_H
 
 #include "f_op/f_op_actor.h"
+#include "dolphin/types.h"
+#include "d/d_com_inf_game.h"
+#include "SSystem/SComponent/c_m3d_g_sph.h"
+#include "SSystem/SComponent/c_cc_s.h"
+#include "SSystem/SComponent/c_phase.h"
+#include "d/d_cc_d.h"
 
 class hitobj_class : public fopAc_ac_c {
+public:
+    inline s32 _create();
+    inline BOOL _delete();
+    inline BOOL _isDelete();
+    inline BOOL _execute();
+    inline BOOL _draw();
```

Comment:
these five inlines don't appear in the debug map.

actors where the class is named like `X_class` are usually coded C-style, so they don't have any member methods, just static functions. so you can remove these and just use `i_this`.